go.mod

```diff
@@ -56,7 +56,7 @@ require (
 
 require (
 	github.com/agiledragon/gomonkey v2.0.2+incompatible
-	github.com/beego/beego v1.12.12
```

Comment:
Will it report error if we don't modify the beego version?

Reply:
When I downgrade the go-sqlite3 version to 1.14, the go mod automatically downgrades the beego version as well, it looks like go-sqlite3 depends on the 1.10 version of beego

Reply:
![112](https://github.com/kubeedge/kubeedge/assets/48212997/2ba7bc55-1d05-4e31-93fc-ff3ad2112253)

Reply:
But I found that the latest version of go-sqlite3 is 1.14.18, I will upgrade it to the latest version

Reply:
How about update beego/beego version to v2？